site/app/templates/grading/electronic/SubmissionPanel.twig

```diff
@@ -108,7 +108,11 @@
             }
         }
         if(!iframe.hasClass("full_panel")){
-            if (!iframe.hasClass('shown')) {
+            if (url_file.substring(url_file.length - 3) === "pdf") {
+                iframe.removeClass('shown');
+                $($($(iframe.parent().children()[0]).children()[0]).children()[0]).removeClass('fa-minus-circle').addClass('fa-plus-circle');
+            }
+            else if (!iframe.hasClass('shown')) {
```

Comment:
I think we can just put a guard around this whole section of `if (!iframe.hasClass("full_panel") && url_file.substring(url_file.length - 3) !== "pdf") {` without changing the body at all and get the desired effect of not changing the button.

The iframe div that exists for PDFs is never used (as they just open straight into full mode with annotator) so we can assume we just want to avoid this whole block.